src/components/ChecklistCard/ChecklistCard.jsx

```diff
@@ -6,9 +6,9 @@ import { Link } from 'react-router-dom'
 function ChecklistCard(props) {
   return (
       <>
-      <Link to={`/checklists/${props.checklist._id}`}>
+      <Link to={`/checklists/${props.checklists._id}`}>
```

Comment:
@vzarate-94 This link looks wrong. What was the reason for switching to `checklists` vs `checklist`?

The typical format would be `collection/doc.id` - ex. `/trips/${props.trip._id}`